docs/entities/serviceclaim.md

```diff
@@ -2,25 +2,30 @@
 
 A Service Claim represents a claim for a Registered Service.
 
+Service Bindings may explicitly request an Application by Name or by LabelSelector. LabelSelector can match more than one resource.
+
```

Comment:
For a better review experience please format the markdown as follow. One line for each sentence, one empty line for creating a new paragraph.

```suggestion
Service Bindings may explicitly request an Application by Name or by LabelSelector.
LabelSelector can match more than one resource.

```